openghg/store/_obscolumn.py

```diff
@@ -108,16 +109,29 @@ def read_file(
         platform = clean_string(platform)
 
         if site is None and satellite is None:
-            raise ValueError("One of 'site' or 'satellite' must be specified")
+            raise ValueError("Value for 'site' or 'satellite' must be specified")
         elif site is not None and satellite is not None:
             raise ValueError("Only one of 'site' or 'satellite' should be specified")
 
         site = clean_string(site)
         satellite = clean_string(satellite)
         domain = clean_string(domain)
+        obs_region = clean_string(obs_region)
+        selection = selection
```

Comment:
```suggestion
```
I think if this is not being updated with `clean_string` this line can be removed.